stock_picking_import_serial_number/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 {
     "name": "Stock Picking Import Serial Numbers",
     "summary": "Import S/N from excel file for incoming pickings",
-    "version": "13.0.1.0.2",
+    "version": "14.0.1.0.2",
```

Comment:
You can reset the version number to `14.0.1.0.0`

Reply:
Thanks @sebalix. Do you know if OCA accepts amend & force-push?

Reply:
Yes